classes/Language.php

```diff
@@ -1263,6 +1263,13 @@ public static function downloadXLFLanguagePack($locale, &$errors = [], $type = s
             return false;
         }
 
+        //Check if response is a zip file
+        if (strpos($content, "\x50\x4b\x03\x04") === false) {
+            $errors[] = Context::getContext()->getTranslator()->trans('Language pack unavailable.', [], 'Admin.International.Notification') . ' ' . $url;
```

Comment:
Hello, are we talking about localization packs or something else? Thanks!

Reply:
Hi,

This file is failing
https://i18n.prestashop-project.org/translations/8.0.1/br-FR/br-FR.zip
thats a translation pack I think (xlf files).

Reply:
Alright :-) the wording is all good to go, thank you!